components/finance/FinanceDateHeader/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button } from '@material-ui/core'
+import { createStyles, makeStyles } from '@material-ui/core/styles'
+import React from 'react'
+
+import DateHelper from '../../../lib/dataHelper'
+
+function createData(date: string, value: number) {
+  return { date, value }
+}
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    todayHeader: {
+      borderRadius: 0,
+      color: 'white',
+      margin: 'auto',
+      width: '10.15rem',
+      background: '#255d6d',
```

Comment:
can this color be set in the material ui theme?

Reply:
I don't see any palette option where I may add new colors apart from updating primary or secondary variations which won't be right.